src/program/snabb_lwaftr/tests/data/binding.table

```diff
@@ -1,7 +1,8 @@
 {
- {'127:2:3:4:5:6:7:128', '178.79.150.233', 1, 100, '8:9:a:b:c:d:e:f'},
- {'127:11:12:13:14:15:16:128', '178.79.150.233', 101, 64000},
- {'127:22:33:44:55:66:77:128', '178.79.150.15', 5, 7000},
- {'127:24:35:46:57:68:79:128', '178.79.150.2', 7800, 7900, '1E:1:1:1:1:1:1:af'},
- {'127:14:25:36:47:58:69:128', '178.79.150.3', 4000, 5050, '1E:2:2:2:2:2:2:af'}
+ {'127:2:3:4:5:6:7:128', '178.79.150.233', {0, 0, 9}, '8:9:a:b:c:d:e:f'},
+ {'127:11:12:13:14:15:16:128', '178.79.150.233', {0, 9, 0}},
```

Comment:
The root cause is indeed a .233 rule. {'127:11:12:13:14:15:16:128', '178.79.150.233', {0, 9, 0}}, excludes only the A=0 ports (0-127), and includes the rest - including 64001, the port in the test, which is indeed using the .233 IPv4 address.